ti/instrumentation/ruby/rspec.go

```
@@ -56,9 +56,13 @@ func (m *rspecRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userA
 		if err != nil {
 			return "", err
 		}
-		err = WriteGemFile(workspace, repoPath)
+		err = AddHarnessRubyAgentToGemfile(repoPath, m.log)
```

we might want to ignore this error.Print a warning and move forward